src/status_im2/common/toasts/events.cljs

```diff
@@ -0,0 +1,50 @@
+(ns status-im2.common.toasts.events
+  (:require
+   [status-im.utils.fx :as fx]
+   [taoensso.encore    :as enc]))
+
+(def ^:private next-toast-id (atom 0))
```

Comment:
can this be moved to app-db ?

Reply:
It can be moved to app-db, just there's no point to do so. It's marked as private and “hide” here deliberately to ensure it can be only changed with `:toasts/create`

Reply:
yeah, i understand, but it is always better to not have local states in events, because its kinda hidden, and not visible in re-frisk

Reply:
done